apps/omg/lib/omg/state/transaction/validator.ex

```diff
@@ -36,27 +37,71 @@ defmodule OMG.State.Transaction.Validator do
           | :unauthorized_spend
           | :utxo_not_found
 
-  @spec can_apply_spend(state :: Core.t(), tx :: Transaction.Recovered.t(), fees :: Fees.optional_fee_t()) ::
-          true | {{:error, exec_error()}, Core.t()}
-  def can_apply_spend(
-        %Core{utxos: utxos} = state,
-        %Transaction.Recovered{signed_tx: %{raw_tx: raw_tx}, witnesses: witnesses} = tx,
+  @type fee_claim_error :: :claiming_unsupported_token | :claiming_more_than_collected
+
+  @type process_error :: exec_error | fee_claim_error
+
+  @doc """
+  can process transaction?
```

Comment:
this doc doesn't tell us much more than the function name, let's either be more helpful here or drop altogether.

A more useful doc would say sth like "Checks, whether at a given state of the ledger, a particular transaction can be applied (!) to it, subject to particular fee requirements"